src/utils/aepp-wallet-communication/rpc/wallet-rpc.ts

```diff
@@ -60,31 +74,37 @@ const REQUESTS = {
     return callInstance(
       'onConnection',
       { name, version },
-      ({ shareNode } = {}) => {
-        client.updateInfo({ status: shareNode ? RPC_STATUS.NODE_BINDED : RPC_STATUS.CONNECTED })
+      ({ shareNode }: { shareNode?: boolean} = {}) => {
+        client.updateInfo({
+          status: shareNode === true ? RPC_STATUS.NODE_BINDED : RPC_STATUS.CONNECTED
+        })
         return {
           result: {
             ...instance.getWalletInfo(),
-            ...(shareNode && { node: instance.selectedNode })
+            ...(shareNode === true && { node: instance.selectedNode })
           }
         }
       },
-      (error) => {
+      (error: any) => {
         client.updateInfo({ status: RPC_STATUS.CONNECTION_REJECTED })
         return { error: ERRORS.connectionDeny(error) }
       }
     )
   },
-  [METHODS.subscribeAddress] (callInstance, instance, client, { type, value }) {
+  [METHODS.subscribeAddress] (
+    callInstance: Function,
+    instance: Ae,
+    client: RpcClient,
+    { type, value }: { type: string, value: string }) {
     // Authorization check
     if (!client.isConnected()) return { error: ERRORS.notAuthorize() }
 
     return callInstance(
       'onSubscription',
       { type, value },
-      async ({ accounts } = {}) => {
+      async ({ accounts }: { accounts?: object } = {}) => {
```

Comment:
```suggestion
      async ({ accounts }: { accounts?: RpcClient['accounts'] } = {}) => {
```

Since `RpcClient['accounts']` is used on the next functions, this could be more suitable here